src/main/kotlin/com/openlattice/data/storage/PostgresDataQueries.kt

```diff
@@ -330,11 +263,11 @@ internal fun selectLinkingEntitySetSql(linkingEntitySetId: UUID): String {
  */
 internal val upsertEntitiesSql = "UPDATE ${IDS.name} " +
         "SET ${VERSIONS.name} = ${VERSIONS.name} || ?, " +
-            "${DataTables.LAST_WRITE.name} = now(), " +
-            "${VERSION.name} = CASE " +
-                "WHEN abs(${IDS.name}.${VERSION.name}) < abs(?) THEN ? " +
-                "ELSE ${IDS.name}.${VERSION.name} " +
-            "END " +
+        "${DataTables.LAST_WRITE.name} = now(), " +
+        "${VERSION.name} = CASE " +
+        "WHEN abs(${IDS.name}.${VERSION.name}) < abs(?) THEN ? " +
+        "ELSE ${IDS.name}.${VERSION.name} " +
+        "END " +
```

Comment:
not of the utmost importance but I think the old formatting here was a bit easier to read

Reply:
Oh, yeah, that was autoformat. Will change it back